src/v2/Apps/ViewingRoom/Routes/Statement/Components/ViewingRoomWorks.tsx

```diff
@@ -50,7 +51,8 @@ export const ViewingRoomWorksFragmentContainer = createFragmentContainer(
   {
     viewingRoom: graphql`
       fragment ViewingRoomWorks_viewingRoom on ViewingRoom {
-        artworksConnection {
+        artworkIDs
```

Comment:
The slight issue here is that I believe artworkIDs include all artworks (published/unpublished) but only published will be viewed on the 'Works' page. So there might be a slight mismatch here. Not sure how that should be properly addressed?

Also not sure how 'unlisted' works will come into play here. Maybe something to discuss tomorrow?